### ISSUE TITLE
TextField titleLabel distance

### ISSUE THREAD

**Issue by xradeon** [CosmicMind/Material#167]
There is no option to set title label distance at layout time.

If you set titleLabelAnimationDistance on viewDidLoad, a TextField with some text will draw his titleLabel at a fixed distance, only after you change some text in it, titleLabel will be drawn taking titleLabelAnimationDistance into account.

**Comment by xradeon** [comment]
My apologies. I tried setting titleLabelAnimationDistance before initializing titleLabel and it worked correctly. Thanks!

**Comment by danieldahan** [comment]
Sounds good. Feel free to open a new issue if you need help.
